<launch>
    <rosparam file="$(find zetabot_main)/param/normar_operater.yaml" command="load" />

    <include file="$(find zetabank_bringup)/launch/zetabank_robot.launch">
        <arg name="rs_serial_no_1" value="1"/>
        <arg name="rs_serial_no_2" value="2"/>
    </include>

    <node pkg="zetabot_main" type="battery_log.py" name="battery_log" respawn="true"/>

    <node pkg="zetabot_main" type="air_log.py" name="air_log" respawn="true"/>

    <node pkg="zetabot_main" type="module_controller.py" name="module_controller" />

    <node pkg="zetabot_main" type="robot_status_for_service.py" name="robot_status_for_service" />

    <node pkg="zetabot_main" type="sonar_filter.py" name="sonar_filter" />

    <include file="$(find zetabank_navigation)/launch/normal_navigation.launch" >
        <arg name="map_file" value="$(find zetabank_navigation)/maps/office2.yaml"/>
    </include>


    <node pkg="zetabot_main" type="initial_pos.py" name="initial_pos_srv" />

    <node pkg="autocharge" type="autocharge.py" name="autocharge_act_srv"/>

    <include file="$(find ai_platform)/launch/ai_platform.launch" />

    <!--autocharge-->
    <!--schedule-->


</launch>
